### PR TITLE
[bug]: default to valid initialValue for checkout form

### LINKED ISSUE #1460: [bug]: Desktop Only  -  Add product to cart > click checkout , without closing checkout right drawer > Sign in to account > Shows error.
**Describe the bug**
Add a product to cart and click on checkout button. Now without closing right drawer, click anywhere on the page then sign in to account from main menu. After login > User gets an error, PFA screen.

**To Reproduce**
1. Go to Venia and add a product to cart.
2. Click on checkout button.
3. Click on "Ship To" link. (Shipping address input form displayed)
4.  Click anywhere in the page to close right drawer. (Just Dont Close right drawer by clicking X icon).
5. Sign in to account via main menu.

**Expected behavior**
User should be logged in without any errors.

**Actual behavior**
User is logged in but gets an error. Which disappears after sometime.

**Screenshots**

### PR DESCRIPTION
**Pull request by dani97**
## Description

Fixes #1460 
## Related Issue

Closes #1460 

### Verification Stakeholders

@dpatil-magento 

### Verification Steps

    Go to Venia and add a product to cart.
    Click on checkout button.
    Click on "Ship To" link. (Shipping address input form displayed)
    Click anywhere in the page to close right drawer. (Just Dont Close right drawer by clicking X icon).
    Sign in to account via main menu.

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.

### CODE REVIEW

File: packages/venia-ui/lib/components/Checkout/addressForm.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ const AddressForm = props => {
 
     const talonProps = useAddressForm({
         fields,
-        initialValues: props.initialValues,
+        initialValues: props.initialValues ? props.initialValues : {},
```
Comment (revanth0212) [inline packages/venia-ui/lib/components/Checkout/addressForm.js]:
```suggestion
        initialValues: props.initialValues || {}
```

Comment (sirugh) [reply packages/venia-ui/lib/components/Checkout/addressForm.js]: This is already set by the `defaultProps`, why is this necessary?

**Comment by dani97** [comment]
@sirugh  on the exact situation reported by @dpatil-magento , null was sent as initialValue to the componenet.

**Comment by dani97** [comment]
> While this does fix the specific issue for the case of the address form it does not for other forms such as payment which you can reproduce by following the repro steps but instead of address form click payment form (and uncheck billing address is same as shipping).
> 
> [Image from Gyazo](https://gyazo.com/344144abe1f9a2620a81696c0ca28c9f)
> 
> Please apply the same fix to the `paymentsForm` component :)

Fixed it 👍

**Pull request merged**